core/storage/exploration/gae_models_test.py

```diff
@@ -316,3 +332,82 @@ def test_get_at_least_editable(self):
             exploration_models.ExpSummaryModel
             .get_at_least_editable('nonexistent_id'))
         self.assertEqual(0, len(exploration_summary_models))
+
+
+class StateIdMappingModelUnitTest(test_utils.GenericTestBase):
+    """Tests for the StateIdMappingModel."""
+
+    def test_create_successfully(self):
+        exploration_models.StateIdMappingModel.create(
+            exp_id='id_0',
+            exp_version=0,
+            state_names_to_ids={},
+            largest_state_id_used=0)
+
+        observed_model = (
+            exploration_models.StateIdMappingModel.
+            get_state_id_mapping_model('id_0', 0))
+
+        self.assertEqual(observed_model.state_names_to_ids, {})
+        self.assertEqual(observed_model.largest_state_id_used, 0)
+
+    def test_create_failed(self):
```

Comment:
Here and elsewhere, make your test name clear about the action and outcomes. E.g.: "test_creating_fails_when_model_already_exists".

See https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests

Reply:
Done.